bindings/python/iota_sdk/types/block/basic.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+from dataclasses import dataclass, field
+from typing import List, Optional, Union
+from iota_sdk.types.block.block import Block, BlockType
+from iota_sdk.types.common import HexStr, json
+from iota_sdk.types.payload import TaggedDataPayload, TransactionPayload
+
+
+@json
+@dataclass
+class BasicBlock(Block):
+    """A `BasicBlock` is the most common type of block used to issue various kinds of payloads such as transactions
+    at the cost of burning Mana.
```

Comment:
Where did you get this doc from?

Reply:
I came up with this because there's nothing in TIP. Open to alternatives.